ui/src/common/containers/CitationSummaryGraphContainer.jsx

```diff
@@ -1,6 +1,92 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import CitationSummaryGraph from '../components/CitationSummaryGraph/CitationSummaryGraph';
 import { convertAllImmutablePropsToJS } from '../immutableToJS';
+import {
+  fetchAuthorPublications,
+  fetchAuthorPublicationsFacets,
+} from '../../actions/authors';
+import { ErrorPropType } from '../propTypes';
+import {
+  CITEABLE_QUERY,
+  PUBLISHED_QUERY,
+  CITEABLE_BAR_TYPE,
+  PUBLISHED_BAR_TYPE,
+} from '../constants';
+
+class CitationSummaryGraphContainer extends Component {
```

Comment:
can we just have function that are called in `mapStateToProps` and `mapDispatchToProps, instead of having a class component that defines a lot of prop types etc.